addToCart/style.css

```diff
@@ -0,0 +1,110 @@
+*{
+    margin:0;
+    padding:0;
+    user-select: none;
+}
+:root{
+    --light-orange: #f6c52d;
+    --dark-orange : #f5aa15;
+    --succes-green: #7acd1f;
+}
+svg{
+    width: 25px;
+    fill: white;
+    height: 35px;
+    padding: 6px 13px;
+    transition: .2s cubic-bezier(0.13, 0.87, 0.34, 1.63) all;
+}
+.wrapper{
+    position:relative;
+}
+body{
+    background: #f7f7f7;
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-family : sans-serif;
+    font-weight: 800;
+    color: white;
+}
+.shoping_button{
+    background: var(--light-orange);
```

Comment:
use background-color to scope to only the property that you want to assign

Reply:
Done, 
i will use it like this from now, i did not knew that so many other values are included in the background prop